templates/main.html

```diff
@@ -8,6 +8,8 @@
   </head>
   <body>
     <h1>GCI Current Leaders</h1>
+    <small id="last-generated" data-time="{{timestamp}}"></small>
```

Comment:
You can put the absolute datetime (.. .. 2017) at build time here,
but still set the relative datetime (.. ago) with client side js.

Like in [coala/community](https://github.com/coala/community/blob/f900af1d42c9c304739953c624024216737637b2/index.html#L14)

It's also nice to have the absolute datetime inside [`<time>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time).
But not mandatory, up to you.